demo/soma_clustering/src/soma_clustering.h

```diff
@@ -38,52 +38,64 @@ enum Substances { kSubstance0, kSubstance1 };
 inline int Simulate(int argc, const char** argv) {
   auto set_param = [](Param* param) {
     // Create an artificial bound for the simulation space
+    param->use_progress_bar = true;
     param->bound_space = Param::BoundSpaceMode::kClosed;
     param->min_bound = 0;
-    param->max_bound = 250;
-    param->unschedule_default_operations = {"mechanical forces"};
+    param->max_bound = 800;
+    // param->unschedule_default_operations = {"mechanical forces"};
   };
 
   Simulation simulation(argc, argv, set_param);
 
   // Define initial model
   auto* param = simulation.GetParam();
-  int num_cells = 20000;
+  int num_cells = 32000;
 
 #pragma omp parallel
   simulation.GetRandom()->SetSeed(4357);
 
   // Define the substances that cells may secrete
   // Order: substance id, substance_name, diffusion_coefficient, decay_constant,
   // and the resolution of the (finite difference) diffusion grid.
-  ModelInitializer::DefineSubstance(kSubstance0, "Substance_0", 50.0, 0.1, 20);
-  ModelInitializer::DefineSubstance(kSubstance1, "Substance_1", 50.0, 0.1, 20);
+  ModelInitializer::DefineSubstance(kSubstance0, "Substance_0", 0.05, 0.001,
+                                    40);
+  ModelInitializer::DefineSubstance(kSubstance1, "Substance_1", 0.05, 0.001,
+                                    40);
 
   int cell_type = 1;
   std::string substance_name = "Substance_0";
 
   auto construct = [&cell_type, &substance_name](const Real3& position) {
     auto* cell = new MyCell(position, cell_type);
     cell->SetDiameter(10);
-    cell->AddBehavior(new Secretion(substance_name));
-    cell->AddBehavior(new Chemotaxis(substance_name, 5));
+    cell->SetMass(0.1);
+    cell->AddBehavior(new Secretion(substance_name, 1));
+    cell->AddBehavior(new Chemotaxis(substance_name, 0.75));
     return cell;
   };
 
   // Construct num_cells/2 cells of type 0
-  ModelInitializer::CreateAgentsRandom(param->min_bound, param->max_bound,
+  ModelInitializer::CreateAgentsRandom(param->max_bound / 2 - 125,
+                                       param->max_bound / 2 + 125,
```

Comment:
Also for the 125 a variable would be good. Like `distance_to_boundary`.

Reply:
I've named this variable `cell_cube_length` and included a comment to explain the need for this:
`// Create a cube of cells in the center of the simulation space`